### ISSUE TITLE
Flexslider slide white space problem

### ISSUE THREAD

**Issue by jopacicdev** [woocommerce/FlexSlider#746]
I'm using Flexslider, and encoutering a strange problem: every slide is shifted to the right, with a portion of white space on the left side of each slide.

When I try to inspect the DOM, I find that temporary fix is to either remove `float: left` or `margin-right: -100%` that's in `li`'s inline style - BUT it breaks the sliding afterwards. What happens is that when you click arrow for the next slide, it shifts the first one down, shows the second, and then hides the first one.

![luxno](https://f.cloud.github.com/assets/2921997/693783/b8d673c8-dc9b-11e2-9098-2a30d9680174.png)

If I don't intervene in anyway with inline styles, slider still has white space to the left, and after I scroll down the page and return up, the slider works as expected, with no problem.

_Note that I don't have this problem on 1920x1080 resolution, but on lower - 1024x768 for example._

What could be the problem? Also, here's a link to live demo: http://goo.gl/RGS4O

**Comment by gaurav1992** [comment]
$('.flexslider').flexslider({
useCSS: false
});
add this line it will work properly

**Comment by jeffikus** [comment]
@gaurav1992 have you tested that in the latest build?

**Comment by gaurav1992** [comment]
@jeffikus Yes i tested with latest flexslider and it work for me.

**Comment by jeffikus** [comment]
@gaurav1992 sorry, I didn't really explain properly, let me try again:

1) Does the latest version - 2.6.0 - have the same issue?
2) IDoes the above code fix the issue in 2.6.0?

Thanks,

**Comment by thomasguillot** [comment]
We're looking into it. The link to the live demo isn't working anymore.

**Comment by JosephP91** [comment]
Hi, I am still having the problem with the big white space under the slides.
It happens when I use images of different size. How can I fix this?
